Tests/ApolloSQLiteTests/CachePersistenceTests.swift

```diff
@@ -59,6 +60,12 @@ class CachePersistenceTests: XCTestCase {
     }
   }
 
+  func testPassInConnection() {
+    let connection = try? Connection()
+    XCTAssertNotNil(connection)
```

Comment:
I would recommend making the test a `throws` test so you don't have to mess around with unwrapping:

```swift
func testPassInConnection() throws {
  let connection = try Connection()
  let cache = try SQLiteNormalizedCache(db: connection)
  // Something that tests that the cache exists
}

Reply:
Oh, sure I can update that.. thanks

Reply:
@designatednerd I updated this to hopefully simplify, but didn't know if there was really a good way to "tests that the cache exists". Instead, I'm simply asserting the creation does not throw.

Reply:
That works too!